ironfish/src/rpc/adapters/tcpAdapter.ts

```diff
@@ -104,7 +104,7 @@ export class TcpAdapter implements IAdapter {
     const reqMap = new Map<string, Request>()
     this.pending.set(connId, { sock: socket, reqs: reqMap })
     socket.on('data', (data) => {
-      this.onClientData(socket, data, reqMap).catch((err) => this.logger.log(err))
+      this.onClientData(socket, data, reqMap).catch((err) => this.logger.error(err))
```

Comment:
Yea `err` here is an `any` value. i'm guessing there's going to be a few other places where this breaks but typescript doesn't catch it. Fortunately it doesn't actually break anything besides the type system. It will still get logged to JSON even if it doesn't conform to the interface